src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionForm.tsx

```diff
@@ -11,16 +11,19 @@ import { Controller, useForm } from 'react-hook-form';
 import { yupResolver } from '@hookform/resolvers/yup';
 import { ProposedSolution } from 'shared';
 import { TextField, Typography, IconButton } from '@mui/material';
+import crypto from 'crypto';
```

Comment:
looks like this is causing some issues:
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/409bf4c2-f055-4a6c-920a-c13c34a464f0)